ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Debian.yml

```diff
@@ -59,19 +65,53 @@
     - ansible_distribution_major_version == "8"
   tags: patch_update
 
-- name: Add Azul Zulu GPG Package Signing Key for x86_64
-  apt_key:
-    url: https://repos.azulsystems.com/RPM-GPG-KEY-azulsystems
-    state: present
-  when:
-    - ansible_architecture == "x86_64"
+# - name: Add Azul Zulu GPG Package Signing Key for x86_64
```

Comment:
Commented out code?

Reply:
Removed comments and added comment explaining azul requirement.